_typos.toml

```
@@ -0,0 +1,15 @@
+[default]
+# allow disabling the next line, e.g:
+#         self.namespaces
+#             // typos:disable-next-line - false positive "typ"
+#             .retain(|ns| ns.typ() != LinuxNamespaceType::Network);
+extend-ignore-re = [
+   "(?Rm)^\\s*//\\s*typos:disable-next-line(\\s.*|\\s*)(\\r|\\n)+[^\\r\\n]*$",
+   # Match 'typ()' when used as a method call
+   "\\w+\\.typ\\(\\)",
+]
```

I don't think the "disable-next-line" thing is being used, right?
```suggestion
extend-ignore-re = [
   # Match 'typ()' when used as a method call
   "\\w+\\.typ\\(\\)",
]
```